pypesto/sampling/diagnostics.py

```diff
@@ -29,8 +31,11 @@ def geweke_test(result: Result,
     # Calculate burn in index
     burn_in = burn_in_by_sequential_geweke(chain=chain,
                                            zscore=zscore)
-    print('Geweke Burn-in index: '+str(burn_in))
 
+    # Log
+    logger.info(f'Geweke burn-in index: {burn_in}')
+
+    # Fill in burn-in value into result
     result.sample_result.burn_in = burn_in
 
-    return result
+    return burn_in
```

Comment:
why changed to return `burn_in` ?

Reply:
that's what the docstring says ;) also makes more sense i think, as result is changed in-place